draft-ietf-quic-multipath.md

```diff
@@ -323,13 +323,28 @@ client on a new path, if the server decides to use the new path,
 the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
 unless it has previously validated that address.
 
-If validation succeed, the client can send non-probing, 1-RTT packets
-on the new paths.  In contrast with the specification in
+If validation succeeds, the client can continue to use the path.
+If validation fails, the client MUST NOT use the path and can
+remove any status associated to the path initation attempt.
+{{Section 9.1 of QUIC-TRANSPORT}} introduces the concept of
+"probing" and "non-probing" frames. When the multipath extension
+is negotiated, the reception of "non-probing"
+packet on a new path needs to be considered as an path initiation
+attempt that does not impact the path status of any existing
+path. Therefore, any frame can be sent on a new path at any time
+as long as the anti-amplification limits
+({{Section 21.1.1.1 of QUIC-TRANSPORT}}) and the congestion control limits are respected.
+
+Further, in contrast with the specification in
 {{Section 9 of QUIC-TRANSPORT}}, the server MUST NOT assume that
-receiving non-probing packets on a new path indicates an attempt
+receiving non-probing packets on a new path with a new Connection ID
+indicates an attempt
 to migrate to that path.  Instead, servers SHOULD consider new paths
 over which non-probing packets have been received as available
-for transmission.
+for transmission. Reception of QUIC packets on a new
+path containing a Connection ID that is already in use on another path
+should be considered as a path migration as further discussed in {{migration}}.
+
```

Comment:
Not a blocking comment, but there is a little issue of how this interacts with PATH_STANDBY and PATH_AVAILABLE. In another PR, we specify that status messages can be sent before a path is created. So we have the case where the peer has received a PATH_STANDBY, and then receives a path validating packet. I would think that the explicit status specified in path status frames always takes priority over the implicit status derived from monitoring the reception of non-probing packets.

Reply:
I don't think there is an issue. Path status relies to a DCID that the other end is using. So if you receive packets on a new path and you want to signal path status for that path, you should send a path status frame together with your path challenge.